platform_versioned_docs/version-23.4.0/data/data-explorer.mdx

```diff
@@ -88,6 +88,18 @@ Data Explorer supports file uploads to your private cloud storage buckets. From
 You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.
 :::
 
+### Download multiple files
+
+You can download multiple files via the user interface or using a script in your cloud provider's CLI. 
+
+Open the cloud bucket that you'd like to download and select the download icon in the **Actions** column. You can choose **Download selected** or **Download selected using code**. The code snippet provided is specific to the cloud provider you've configured. You may be prompted to authenticate during the download process.
+
+Refer to your cloud provider's documentation for troubleshooting credential-related issues:
```

Comment:
```suggestion
Open the cloud bucket that you'd like to download and select the download icon in the **Actions** column. You can choose **Download selected** or **Download selected using code**. 

The code snippet provided is specific to the cloud provider you've configured. You may be prompted to authenticate during the download process. Refer to your cloud provider's documentation for troubleshooting credential-related issues:
```
Mainly because it seems likely that we'll need to add a separate paragraph to explain the other option ("Download selected"), to which the provider cli instructions don't apply

Reply:
Good point. This will be added when I can test out the feature.